src/Builder/ClassModifier/ClassInstance/Wire/WireModifier.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Builder\ClassModifier\ClassInstance\Wire;
+
+use RuntimeException;
+use SprykerSdk\Integrator\Builder\Checker\ClassMethodCheckerInterface;
+use SprykerSdk\Integrator\Builder\ClassModifier\AddVisitorsTrait;
+use SprykerSdk\Integrator\Builder\ClassModifier\CommonClass\CommonClassModifierInterface;
+use SprykerSdk\Integrator\Builder\Finder\ClassNodeFinderInterface;
+use SprykerSdk\Integrator\Transfer\ClassInformationTransfer;
+use SprykerSdk\Integrator\Transfer\ClassMetadataTransfer;
+
+class WireModifier implements WireModifierInterface
```

Comment:
Could you please provide more explicit naming for all wire and unwire class modifiers.

Reply:
Thanks ++